### PR TITLE
Feature/component bottomsheet

### PR DESCRIPTION
**Pull request by RomanKondr**

### CODE REVIEW

File: src/components/ui/bottomsheet.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import { cn } from '../../lib/utils';
+
+interface BottomSheetProps {
+  visible: boolean;
+  onClose: () => void;
+  children?: React.ReactNode;
+  containerClassName?: string;
+  backgroundColor?: string;
+  borderRadius?: string;
+  height?: number;
```
Comment (Lomsss) [inline src/components/ui/bottomsheet.tsx:21]: You do not need it, if you are using cn()

File: src/components/ui/bottomsheet.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+const screenHeight = Dimensions.get('window').height;
+
+const StyledView = styled(View);
```
Comment (Lomsss) [inline src/components/ui/bottomsheet.tsx:26]: You do not need it

File: src/components/ui/bottomsheet.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    <Modal transparent visible={visible} animationType="none">
+      <TouchableWithoutFeedback onPress={animateModalOut}>
+        <Animated.View
+          style={[
+            StyleSheet.absoluteFillObject,
+            {
+              backgroundColor: 'rgba(0, 0, 0, 0.5)',
+              opacity: backgroundOpacity,
+            },
+          ]}
```
Comment (Lomsss) [inline src/components/ui/bottomsheet.tsx:108]: Use className instead

**Review by Lomsss — CHANGES_REQUESTED** [review]

File: src/components/ui/actionoptions.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  GestureResponderEvent,
+} from 'react-native';
+
+import { cn } from '../../lib/utils';
+
+interface ActionButtonProps {
+  icon?: React.ReactNode;
+  label: string;
+  labelSize?: number;
+  labelColor?: string;
```
Comment (Lomsss) [inline src/components/ui/actionoptions.tsx:15]: Use className for cases like this

File: src/components/ui/actionoptions.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  labelSize = 18,
+  labelColor = 'black',
+  onPress,
+  containerStyle = {},
+  contentStyle = {},
+}) => {
+  return (
+    <TouchableOpacity
+      onPress={onPress}
+      style={[cn('flex-row items-center justify-start py-3'), containerStyle]}
```
Comment (Lomsss) [inline src/components/ui/actionoptions.tsx:33]: Do not use cn(), if you are not going to pass a variable class to it

**Review by Lomsss — CHANGES_REQUESTED** [review]

File: src/components/ui/bottomsheet.tsx (hunk 1 above)
Comment (RomanKondr) [reply src/components/ui/bottomsheet.tsx:21]: cn don't work for that

File: src/components/ui/bottomsheet.tsx (hunk 2 above)
Comment (RomanKondr) [reply src/components/ui/bottomsheet.tsx:26]: corrected

File: src/components/ui/bottomsheet.tsx (hunk 3 above)
Comment (RomanKondr) [reply src/components/ui/bottomsheet.tsx:108]: corrected

File: src/components/ui/actionoptions.tsx (hunk 4 above)
Comment (RomanKondr) [reply src/components/ui/actionoptions.tsx:15]: cn don't work for that

**Pull request merged**